api/nakadi-event-bus-api.yaml

```diff
@@ -1152,3 +1487,99 @@ definitions:
           should have a description.
     required:
       - publishing_status
+
+parameters:
+  EventTypeName:
+    name: name
+    in: path
+    description: EventType name to get events about
+    type: string
+    required: true
+
+  BatchLimit:
+    name: batch_limit
+    in: query
+    description: |
+      Maximum number of `Event`s in each chunk (and therefore per partition) of the stream.
+
+      * If 0 or unspecified will buffer Events indefinitely and flush on reaching of
+      `batch_flush_timeout`.
+    type: integer
+    format: int32
+    required: false
+    default: 1
+
+  StreamLimit:
+    name: stream_limit
+    in: query
+    description: |
+      Maximum number of `Event`s in this stream (over all partitions being streamed in this
+      connection).
+
+      * If 0 or undefined, will stream batches indefinitely.
+
+      * Stream initialization will fail if `stream_limit` is lower than `batch_limit`.
+    type: integer
+    format: int32
+    required: false
+    default: 0
+
+  BatchFlushTimeout:
+    name: batch_flush_timeout
+    in: query
+    description: |
+      Maximum time in seconds to wait for the flushing of each chunk (per partition).
+
+      * If the amount of buffered Events reaches `batch_limit` before this `batch_flush_timeout`
+      is reached, the messages are immediately flushed to the client and batch flush timer is reset.
+
+      * If 0 or undefined, will assume 30 seconds.
+    type: number
+    format: int32
+    required: false
+    default: 30
+
+  StreamTimeout:
+    name: stream_timeout
+    in: query
+    description: |
+      Maximum time in seconds a stream will live before being interrupted.
+      If 0 or unspecified will stream indefinitely.
+
+      If this timeout is reached, any pending messages (in the sense of `stream_limit`) will be flushed
+      to the client.
+
+      Stream initialization will fail if `stream_timeout` is lower than `batch_flush_timeout`.
+    type: number
+    format: int32
+    required: false
+    default: 0
+
+  XFlowId:
+    name: X-Flow-Id
+    in: header
+    description: |
+      The flow id of the request, which is written into the logs and passed to called services. Helpful
+      for operational troubleshooting and log analysis.
+    type: string
+    format: flow-id
+
+  SubscriptionId:
+    name: subscription_id
+    in: path
+    description: Id of subscription.
+    type: string
+    format: uuid
+    required: true
+
+  StreamKeepAliveLimit:
```

Comment:
I was wondering what a keepalive is. This has a specific meaning in TCP/IP, but I've also heard dummy batches on the stream called keepalives :)